### DISCUSSION TITLE
Unable to rejoin cluster after crashed osd purge

### DISCUSSION THREAD

**Discussion by df-bingo** [rook/rook#9929] [Q&A]
There's nothing I can do about it, please help me.
There were osd crashbackoffs constantly, and then the osd was down and out. The strange thing is that the crashed osd are all from the k8s-82 node. I don't know what happened to this node? I cleaned up the osd according to the process of the [document,](https://rook.io/docs/rook/v1.8/ceph-osd-mgmt.html#remove-an-osd) but those osd couldn't rejoin the cluster.
The osd-prepare-k8s-82 started to report an error, and it took a while. After that, it exited. From the log of [rook-ceph-osd-prepare-k8s-82--1-xxf5c.log](https://github.com/rook/rook/files/8302631/rook-ceph-osd-prepare-k8s-82--1-xxf5c.log), it was because an error was reported when processing osd.15,. Osd-prepare-k8s -82 exit with an error, all the hard disks on the k8s-82 node are not added to the cluster, but are formatted as LVs. Although osd-prepare-k8s-81 succeeds([rook-ceph-osd-prepare-k8s-81--1-kmdzh.log](https://github.com/rook/rook/files/8302630/rook-ceph-osd-prepare-k8s-81--1-kmdzh.log)), the two hard disks on the k8s-81 node are not Join the cluster.

[ceph_osd_tree.txt](https://github.com/rook/rook/files/8302629/ceph_osd_tree.txt)

![Screenshot from 2022-03-18 15-43-17](https://user-images.githubusercontent.com/84017898/158959982-e0d72637-0efb-4c9e-82cf-a06057cefea9.png)

**Comment by travisn** [comment]
After you purged the OSDs, did you [wipe the disks](https://rook.github.io/docs/rook/v1.8/ceph-teardown.html#zapping-devices)? You will need to wipe them to get clean OSDs created and rejoin the cluster. Otherwise, the operator will try to keep re-creating OSDs, which won't be able to reconnect.

**Reply by df-bingo** [reply]
Of course, I wiped the disks, [that's](https://rook.github.io/docs/rook/v1.8/ceph-teardown.html#zapping-devices) how I did it .
